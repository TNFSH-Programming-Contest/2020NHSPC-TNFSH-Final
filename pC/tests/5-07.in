100 197 55
48 55 781599
55 67 447003
48 67 936143
64 48 211226
67 64 271047
67 80 46949
64 80 928560
33 64 144273
80 33 526889
28 80 686145
33 28 302195
33 66 123936
28 66 526189
99 28 974281
99 66 697138
50 66 197262
99 50 861056
99 39 607637
50 39 671955
50 82 910463
39 82 870642
39 62 233704
82 62 162967
82 52 444328
62 52 274920
60 62 506565
60 52 749252
41 52 888321
41 60 797507
92 60 31937
41 92 443744
13 41 294614
13 92 428435
92 2 150191
2 13 56374
13 27 654395
2 27 17510
2 10 739357
10 27 104497
27 91 780727
91 10 719670
77 10 263371
77 91 528595
90 91 809285
90 77 197638
54 77 198324
90 54 367323
90 87 711197
54 87 335293
54 97 342825
97 87 788175
96 87 308058
96 97 619368
97 3 489856
3 96 611112
70 96 425260
70 3 543891
3 30 268999
30 70 479512
100 70 252497
100 30 854534
22 30 460039
100 22 826812
42 100 644630
22 42 853582
93 22 349685
42 93 425802
42 86 410946
86 93 810623
46 93 307348
46 86 151117
24 86 842183
46 24 469985
46 44 819932
44 24 286930
43 24 159452
43 44 419256
15 44 478473
43 15 412922
43 23 600986
23 15 586204
15 75 446759
23 75 829554
9 23 284001
9 75 109511
40 75 887097
40 9 129096
9 19 4534
40 19 172121
31 40 870538
31 19 836647
19 38 891045
38 31 675082
31 85 842393
38 85 427788
35 38 206308
35 85 594401
85 73 55312
35 73 322254
35 29 853318
73 29 264157
73 6 981661
6 29 804095
29 34 376461
34 6 309537
6 68 477568
68 34 453831
21 34 916462
21 68 357302
18 68 728543
21 18 40902
21 65 469500
65 18 992817
18 37 165875
37 65 89850
61 65 417894
61 37 696700
37 95 835944
61 95 719621
61 58 213070
95 58 797305
5 95 242561
5 58 262603
58 12 182046
5 12 766243
71 5 13214
12 71 689977
12 78 372615
71 78 338577
71 20 814402
78 20 648993
8 78 701421
20 8 37671
59 20 29304
8 59 223909
8 7 916725
59 7 592439
14 59 660890
7 14 755112
7 26 362717
14 26 833430
74 14 350312
26 74 299774
26 83 965910
83 74 288914
74 84 734581
83 84 970806
83 16 238348
16 84 233215
81 84 177947
81 16 638188
72 16 558592
81 72 192199
81 11 856129
11 72 557894
63 72 832624
11 63 249905
94 11 399296
63 94 774546
89 63 903396
94 89 939323
94 49 395099
89 49 19806
89 53 789887
53 49 552228
98 49 603924
98 53 814176
45 53 90003
45 98 266109
1 98 220096
45 1 983532
69 45 710828
69 1 629125
1 57 606553
69 57 817277
88 69 626169
88 57 410820
55 51 3
51 39 3
25 55 2
25 12 3
55 32 3
32 39 3
55 17 5
39 17 3
56 55 1
56 60 2
79 55 1
79 29 3
55 76 5
19 76 2
55 47 3
69 47 2
4 55 4
4 52 2
36 55 2
59 36 2
