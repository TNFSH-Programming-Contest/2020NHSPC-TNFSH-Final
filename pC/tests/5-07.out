1887390 728524 15985140 4 1778616 2412297 1777329 267916 13623 1824828 15032213 10 1192525 1982682 2804055 9230808 5 6669145 14 87924 5171708 19583982 1154168 6293880 2 3820640 998205 3540291 8 20542643 2509962 3 2586969 1129395 2345660 2 5417190 2673156 12 516384 1902736 18416872 4447640 6235704 2132499 10632755 3 781599 6765320 2015883 3 12 3203344 6126664 0 1 2451846 546153 8 6 1580484 701130 14964714 1436100 4065075 3476892 447003 3321184 10 20178884 2069946 14679420 792483 4044824 456208 5 3972563 1117860 1 987904 11542404 1333002 6500600 8728935 1277892 13238141 9910134 1878522 7953615 6121176 5882208 95820 14610504 10488978 3436075 14091840 9977922 3396904 1822929 25043292 
